### ISSUE TITLE
Service status: colour conveys meaning.

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_services#306]
The service statuses have a type set - Limited service, Revised service, Normal service. 

In the template this applies one of the classes `alert-danger` `alert-info` or `alert-success` - also used in alert banners and WYSWYG text styles I think. 

This, by default changes the colour to red, blue or green. 

If this information conveys meaning, then this should be available in another way also. Either in text and/or an icon. as discussed https://github.com/localgovdrupal/localgov_alert_banner/issues/345

**Comment by msayoung** [comment]
@willguv - are we likely to deprecate service status content type? 
Should I work on this issue, or leave it?

**Comment by willguv** [comment]
Hi Maria, I'd like to keep and use more. Is this issue a lot of effort please?
